### PR TITLE
[TECHNICAL SUPPORT] LPS-124514 Each form entry has its respective version of form instance

### PR DESCRIPTION
**Pull request by rodrigopaulino**
Relevant ticket: https://issues.liferay.com/browse/LPS-124514

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_7_3/UpgradeDDMFormInstanceReport.java [hunk 1]
```diff
@@ -70,16 +69,25 @@ public UpgradeDDMFormInstanceReport(
...
-		sb1.append("DDMContent inner join DDMFormInstanceRecordVersion on ");
+		sb1.append("DDMFormInstanceRecord.formInstanceRecordId, ");
+		sb1.append("DDMStructureVersion.definition from DDMContent inner ");
+		sb1.append("join DDMFormInstanceRecordVersion on ");
 		sb1.append("DDMContent.contentId = ");
 		sb1.append("DDMFormInstanceRecordVersion.storageId inner join ");
 		sb1.append("DDMFormInstanceRecord on ");
 		sb1.append("DDMFormInstanceRecord.formInstanceRecordId = ");
-		sb1.append("DDMFormInstanceRecordVersion.formInstanceRecordId where ");
+		sb1.append("DDMFormInstanceRecordVersion.formInstanceRecordId join ");
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_7_3/UpgradeDDMFormInstanceReport.java:82]: @rodrigopaulino, let's use `inner join` to keep consistency with existent query code. 🙂

**Review by natocesarrego — CHANGES_REQUESTED** [review]

**Comment by natocesarrego** [comment]
@marcosapmf, can you please review this one once you've created this Upgrade Step?

Thanks in advance.

**Comment by marcosapmf** [comment]
@natocesarrego @rodrigopaulino, It looks good to me :)

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_7_3/UpgradeDDMFormInstanceReport.java (hunk 1 above)
Comment (rodrigopaulino) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_7_3/UpgradeDDMFormInstanceReport.java:82]: You got it.

**Pull request closed without merging**